configs/template_config.yaml

```diff
@@ -44,12 +44,41 @@ models_path: "/YOUR/PATH/HERE/data/models"
 # TensorBoard Path
 tboard_path: ~
 
+# Checkpoint Path: path to checkpoint to load for generation/inference
+checkpoint_path: ~
```

Comment:
If we are to implement a `checkpoint_path` argument, shouldn't this be implemented everywhere (ex: the normal `train_model.py`)?

Reply:
The purpose of this parameter is that, once you've trained a model and want to do inference or generation, you put into this field the path to the checkpoint that you want to use for those tasks. This isn't used in the training scripts.